runners/spark/src/main/java/org/apache/beam/runners/spark/examples/WordCount.java

```diff
@@ -47,14 +47,16 @@
   public static class ExtractWordsFn extends DoFn<String, String> {
     private final Counter emptyLines = Metrics.counter(ExtractWordsFn.class, "emptyLines");
 
+    private static final String TOKENIZER_PATTERN = "[^\\p{L}]+";
```

Comment:
Could you import `import org.apache.beam.examples.common.ExampleUtils;` instead?